README.md

```diff
@@ -12,7 +12,42 @@ pip install ./
 
 ---
 
-For some examples of usage, take a look at the integration tests.
+### Examples
+
+To display a bitmap image:
+
+```python
+from IT8951.display import AutoEPDDisplay
+from IT8951 import constants
+from PIL import Image
+
+display = AutoEPDDisplay(vcom=-2.48)
+
+# Make display clear
+display.clear()
+
+display.frame_buf.paste(0xFF, box=(0, 0, display.width, display.height))
+img = Image.open('IT8951/test/integration/images/sleeping_penguin.png')
+dims = (display.width, display.height)
+img.thumbnail(dims)
+
+# Align image with bottom of display
+paste_coords = [dims[i] - img.size[i] for i in (0,1)]
+display.frame_buf.paste(img, paste_coords)
+
+# Display the image
+display.draw_full(constants.DisplayModes.GC16)
+```
+
+To display some text:
+
+```python
+from IT8951.display import AutoEPDDisplay
+from IT8951 import constants
+
```

Comment:
@GregDMeyer is there a quick and easy way here we show how to get some text on the display without using font files?

Reply:
:( unfortunately there is not, as far as I know.

Reply:
OK no worries. Leave it with me for a few days and I'll write something.

Reply:
OK I changed the code to display a polygon. This way there's no need to install any fonts.